docs/src/main/asciidoc/datastore.adoc

```diff
@@ -374,19 +372,61 @@ There are three ways to represent relationships between entities that are descri
 ==== Embedded Entities
 
 Fields whose types are also annotated with `@Entity` are converted to `EntityValue` and stored inside the parent entity.
+
+Here is an example of Cloud Datastore entity containing an embedded entity in JSON:
+
+[source, json]
+----
+{
+  "name" : "Alexander",
+  "age" : 50,
+  "children" : [ {
```

Comment:
I think a simpler example would be where it's not a collection, but just an embedded singular entity.